makefiles/Makefile.cloudgov-docs

```diff
@@ -20,6 +20,3 @@ include ~/shared.mk
 get-build-dependencies: 
 	@curl https://raw.githubusercontent.com/mongodb/docs-worker-pool/meta/publishedbranches/cloudgov-docs.yaml > ${REPO_DIR}/published-branches.yaml
 
-
-html: ## Builds this branch's HTML under build/<branch>/html
-	giza make html
```

Comment:
I removed this rule and several in other files with this same format; however, there is no next-gen-html rule in this and some of the other makefiles to replace it. Is that of concern?

Reply:
Nope! That shouldn't be of concern. These makefiles have an `include ~/shared.mk` statement at the top. The [`shared.mk`](https://github.com/mongodb/docs-worker-pool/blob/meta/makefiles/shared.mk) file contains the next-gen rules.